common/source/docs/common-geofencing-landing-page.rst

```diff
@@ -48,19 +41,233 @@ Inclusion and Exclusion Fences are defined easily in Mission Planner under the P
 |   - FENCE_ALT_MIN defines min altitude|                    |   X       |          |         |
 +---------------------------------------+--------------------+-----------+----------+---------+
 
+.. _fence_breach_actions:
 
-Detailed Information
+Fence Breach Actions
 ====================
+[site wiki="copter,rover"]
+Rover Actions
+-------------
+
+Rover has no altitude actions, but if it reaches the Cylindrical fence boundary in either Steering or Acro modes, it will simply stop and not move beyond it. In other modes, it will execute the :ref:`FENCE_ACTION<FENCE_ACTION>`.
+
+Copter Actions
+--------------
+
+Copter will stop increasing altitude at :ref:`FENCE_ALT_MAX<FENCE_ALT_MAX>` in Loiter, PosHold, and AltHold modes.
+
+In Loiter mode, it will stop at the horizontal boundaries.
+
+In other modes, it will execute the :ref:`FENCE_ACTION<FENCE_ACTION>` below if the altitude or boundary is reached.
```

Comment:
In fact it will execute these actions in any mode .. it's difficult but still possible to breach the fence in Loiter mode.

Reply:
will reword